scripts/edd-get-appdata

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```

Comment:
i followed this: https://docs.python.org/2/distutils/setupscript.html#installing-scripts
couldn't really find a better doc, happy to read more.